discord/app_commands/commands.py

```diff
@@ -48,6 +48,8 @@
 import re
 from copy import copy as shallow_copy
 
+from discord.flags import AppCommandContext
```

Comment:
```suggestion
from .flags import AppCommandContext
```

and move it down to where the others are